lib/workers/pr/body/index.ts

```diff
@@ -13,7 +13,9 @@ import { getPrUpdatesTable } from './updates-table';
 function massageUpdateMetadata(config: BranchConfig): void {
   config.upgrades.forEach((upgrade) => {
     /* eslint-disable no-param-reassign */
-    const { homepage, sourceUrl, sourceDirectory, changelogUrl } = upgrade;
+    const { sourceUrl, sourceDirectory } = upgrade;
+    const homepage = upgrade.homepage;
+    const changelogUrl = upgrade.changelogUrl;
```

Comment:
Why do these changes help?

Reply:
`BranchConfig` -> `RenovateConfig` -> `Record<string, unknown>`

Reply:
I still don't get why the change in syntax affects anything.

Reply:
Ah, now we can revert it